code/__HELPERS/areas.dm

```diff
@@ -117,17 +122,54 @@ GLOBAL_LIST_INIT(typecache_powerfailure_safe_areas, typecacheof(/area/station/en
 		newA = new area_choice
 		newA.setup(str)
 		newA.has_gravity = oldA.has_gravity
+		require_area_resort() //new area registered. resort the names
 	else
 		newA = area_choice
 
+	//we haven't done anything. let's get outta here
+	if(newA == oldA)
+		to_chat(creator, span_warning("The area remains unchanged"))
+		return
+	//when expanding one area into another we don't want to merge their apcs. Display warning to user to dismantle the old area apc inorder to proceed
+	if(!isnull(newA.apc) && !isnull(oldA.apc))
+		to_chat(creator, span_warning("Current & new area both have APCs!!. Dismantle any one apc before proceeding."))
```

Comment:
```suggestion
		creator.balloon_message(creator, "Conflicting APCs, remove one!")
```

Reply:
balloon alerts have to be all undercase (at least for `Conflicting`). might be better to do something like `too many APCs!`, that seems informative enough to me IMO. (acronyms being capitalized appear to be fine looking at the code, for examples like `TK` for telekinesis and `DNA`)

Reply:
How about "too many conflicting APCS?" I think that makes both sides happy

Reply:
Got a syntax error. its `balloon_alert()` not `balloon_message()`